### ISSUE TITLE
razor, cshtml files

### ISSUE THREAD

**Issue by moreiraio** [jmederosalvarado/roslyn.nvim#32]
Hi,

quick question, does this support razor cshtml files?

**Comment by moreiraio** [comment]
Hi, I've managed to make the razor lsp work and attach.

**Comment by VKG147** [comment]
@moreiraio could you perhaps share your configuration that you got working with the razor lsp? I'd greatly appreciate it :)

**Comment by moreiraio** [comment]
Hey, yes:
https://github.com/moreiraio/razor.nvim

It's a draft yet just to make it work, I want it to evolve to a plugin.

I've been able to intercept the lsp messages between vscode and the server.

If more people want to join in making the c#/.net experience great in neovim that would be great

**Comment by VKG147** [comment]
Thank you! I'm all for making the C# experience better for neovim users. Coming from visual studio I'm still very new to neovim and lua, hopefully one day I can help the effort.

**Comment by irowi** [comment]
Much appreciated! I'm also all for making the C#/.NET experience better for neovim users. Razor is the missing piece, atleast for me to fully go all in. But before i try to help i want to make sure there isn't any license concerns regarding rzls lsp with the vscode license.

**Comment by InfinytRam** [comment]
@moreiraio nice!!! are you able to provide lua code example to demonstrate how to integrate it with lspconfig using Lazy package manger?  

I'm new to neovim, but from looking at your source, I dont really understand how to configure it to my neovim.

**Comment by InfinytRam** [comment]
@moreiraio , nice!! are you able to provide docs on how to set it up?

**Comment by marcinjahn** [comment]
I suggest you have a look at this fork: https://github.com/seblj/roslyn.nvim
This repo seems to be dead, while the other one is accepting PRs and seems to be growing.

**Comment by InfinytRam** [comment]
Thanks @marcinjahn, does that plugin support LSP for razor/cshtml files?

**Comment by marcinjahn** [comment]
Nope, but if any of you intend to add such support, I guess there’s much bigger chance for such a PR to be merged there than here :)
